.github/workflows/e2e.yaml

```diff
@@ -1,6 +1,9 @@
 name: e2e
 on:
   pull_request:
+    paths:
+      - '**.go'
+      - '.github/workflows/e2e.yml'
```

Comment:
also go.{mod,sum} files?

Reply:
Only on the root level? Or all go.{mod|sum}?

Reply:
probably all files for safety.